### PR TITLE
Add example of Shared::Field with selects

### PR DESCRIPTION
**Pull request by paulcsmith**

### CODE REVIEW

File: src/actions/guides/frontend/html_forms.cr [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Guides::Frontend::HtmlForms < GuideAction
+  ANCHOR_SHARED_COMPONENTS = "shared-components"
```
Comment (jwoertink) [inline src/actions/guides/frontend/html_forms.cr:2]: All of the others we've prefixed with `perma-`.. I don't remember why though.

File: src/actions/guides/frontend/html_forms.cr [hunk 2]
```diff
@@ -341,6 +344,22 @@ class Guides::Frontend::HtmlForms < GuideAction
...
 
+    ### Using selects with `Shared::Field` component
+
+    Here is how you would use `select_input` with a `Shared::Field` or other
+    field component.
+
+    ```
+    mount Shared::Field.new(op.car_make) do |input_html|
+      input_html.select_input append_class: "select-input" do
+      options_for_select op.car_make, options_for_cars
```
Comment (jwoertink) [inline src/actions/guides/frontend/html_forms.cr]:
```suggestion
        options_for_select op.car_make, options_for_cars
```

File: src/actions/guides/frontend/html_forms.cr (hunk 1 above)
Comment (paulcsmith) [reply src/actions/guides/frontend/html_forms.cr:2]: Probably me overthinking things :P I'll add perma to match the others

**Comment by paulcsmith** [comment]
Thanks for the review!

**Pull request merged**